pybleau/app/plotting/plot_style.py

```diff
@@ -390,8 +388,8 @@ def _get_specific_view_elements(self):
 
     # Traits listener methods -------------------------------------------------
 
-    @on_trait_change("color_palette, renderer_styles[]")
-    def update_renderer_colors(self):
+    @observe("color_palette", post_init=True)
+    def update_renderer_colors(self, event):
```

Comment:
What is the purpose of adding the `post_init=True`? Also, you don't seem to be using this `event` argument. Remove?

Reply:
I did some testing, and the `post_init` isn't necessary now.

As I understand it for `observe`, the handler has to have `event` in the arguments list [link](https://docs.enthought.com/traits/traits_user_manual/notification.html#observe-handler).

> What is the purpose of adding the post_init=True? Also, you don't seem to be using this event argument. Remove?